src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int downSide;
+    private int topSide;
+    private int lateralSide;
```

Comment:
3rd side should be height, and fix the area formula accordingly
```suggestion
    private int height;
```